convert2rhel/actions/system_checks/is_loaded_kernel_latest.py

```diff
@@ -129,6 +137,16 @@ def run(self):  # pylint: disable= too-many-return-statements
             )
             return
 
+        self.add_message(
+            level="WARNING",
+            id="UNSUPPORTED_SKIP_KERNEL_CURRENCY_CHECK_DETECTED",
+            message=(
+                "Detected 'CONVERT2RHEL_UNSUPPORTED_SKIP_KERNEL_CURRENCY_CHECK' environment variable, we will skip "
+                "the %s comparison.\n"
+                "Beware, this could leave your system in a broken state. " % package_to_check
+            ),
+        )
```

Comment:
I think this is in the wrong part of the code, it should be on line 82

Reply:
I will move this